contracts/src/v0.8/automation/v2_1/AutomationUtils2_1.sol

```diff
@@ -10,6 +10,21 @@ import {Log} from "../interfaces/ILogAutomation.sol";
  * and tests because generated wrappers are made available
  */
 
+/**
+ * @notice encode the checkErrorHandler payload which will be used in executeCallback
+ * @dev 0x254b90dc is Keccak hash of the checkErrorHandler function
+ * function checkErrorHandler(
+ *   bytes[] memory values,
+ *   bytes memory extraData
+ *  ) external view returns (bool upkeepNeeded, bytes memory performData);
+ */
+function encodeCheckErrorHandler(
```

Comment:
@infiloop2 / @amirylm  : I am not very sure if this "free function" (not part of contract or library) is invokable from offchain code. I think so, but let me know otherwise.